app/code/Magento/CatalogImportExport/Model/Import/Product/Option.php

```diff
@@ -1287,6 +1281,7 @@ protected function _importData()
                     $this->_rowProductId = $this->_productsSkuToId[$rowData[self::COLUMN_SKU]];
                     if (array_key_exists('custom_options', $rowData)
                         && (
+                            $rowData['custom_options'] === null ||
                             trim($rowData['custom_options']) === '' ||
                             trim($rowData['custom_options']) === $this->_productEntity->getEmptyAttributeValueConstant()
                         )
```

Comment:
Hello @SilinMykola 
What do you think about suggestion below?

```suggestion
                    if (isset($rowData['custom_options'])
                        && (
                            trim($rowData['custom_options']) === '' ||
                            trim($rowData['custom_options']) === $this->_productEntity->getEmptyAttributeValueConstant()
                        )
```

Reply:
Hello @andrewbess 
Thanks for your review. I made changes, please check.